server/db.sql

```diff
@@ -0,0 +1,282 @@
+7	Long Neck	Dr. Sarah Brown	sarah.brown@example.com	4	2023-09-20 20:28:26.436685
+8	Spots	Dr. Daniel Wilson	daniel.wilson@example.com	4	2023-09-20 20:28:26.436685
+9	Snowball	Dr. Rachel Lee	rachel.lee@example.com	5	2023-09-20 20:28:26.436685
+10	Iceberg	Dr. Robert Clark	robert.clark@example.com	5	2023-09-20 20:28:26.436685
+11	Bamboo	Dr. Jessica Taylor	jessica.taylor@example.com	6	2023-09-20 20:28:26.436685
+12	Panda Jr.	Dr. Matthew Green	matthew.green@example.com	6	2023-09-20 20:28:26.436685
+16	Leo	Dr Dao	drdao@gmail.com	1	2023-09-22 15:13:48.742064
+17	Cow	Cathy	\N	2	2023-09-25 15:52:24.599857
+18	Cow	Cathy	\N	2	2023-09-25 15:52:59.19844
+19	Cow	Cathy	\N	2	2023-09-25 15:53:19.374226
+\.
+
+
+--
+-- Data for Name: sightings; Type: TABLE DATA; Schema: public; Owner: tpl1122_1
+--
+
+COPY public.sightings (id, sighting_datetime, individual_id, sighting_location, is_healthy, scientist_name, scientist_email, created_at) FROM stdin;
+1	2023-09-10 08:00:00	1	Savannah	t	Dr. Sarah Johnson	sarah.johnson@example.com	2023-09-20 20:28:40.587487
+2	2023-09-12 14:30:00	1	Savannah	f	Dr. David Wilson	david.wilson@example.com	2023-09-20 20:28:40.587487
+3	2023-09-11 10:15:00	2	Savannah	t	Dr. Lisa Anderson	lisa.anderson@example.com	2023-09-20 20:28:40.587487
+4	2023-09-13 16:45:00	2	Savannah	f	Dr. Michael Wilson	michael.wilson@example.com	2023-09-20 20:28:40.587487
+5	2023-09-10 09:30:00	3	Jungle	t	Dr. Emily Davis	emily.davis@example.com	2023-09-20 20:28:40.587487
+6	2023-09-12 11:45:00	3	Jungle	f	Dr. John Smith	john.smith@example.com	2023-09-20 20:28:40.587487
+7	2023-09-11 08:45:00	4	Jungle	t	Dr. Jessica Taylor	jessica.taylor@example.com	2023-09-20 20:28:40.587487
+8	2023-09-13 12:00:00	4	Jungle	f	Dr. Mary Johnson	mary.johnson@example.com	2023-09-20 20:28:40.587487
+9	2023-09-10 15:20:00	5	Arctic	t	Dr. Daniel Wilson	daniel.wilson@example.com	2023-09-20 20:28:40.587487
+10	2023-09-12 09:30:00	5	Arctic	f	Dr. Robert Clark	robert.clark@example.com	2023-09-20 20:28:40.587487
+11	2023-09-11 13:40:00	6	Arctic	t	Dr. Rachel Lee	rachel.lee@example.com	2023-09-20 20:28:40.587487
+12	2023-09-13 17:15:00	6	Arctic	f	Dr. Matthew Green	matthew.green@example.com	2023-09-20 20:28:40.587487
+\.
+
+
+--
+-- Data for Name: species; Type: TABLE DATA; Schema: public; Owner: tpl1122_1
+--
+
+COPY public.species (id, common_name, scientific_name, estimated_population, conservation_status_code, created_at) FROM stdin;
+1	Lion	Panthera leo	20000	EN	2023-09-20 20:28:11.297703
+2	Elephant	Loxodonta africana	415000	VU	2023-09-20 20:28:11.297703
+3	Tiger	Panthera tigris	3900	EN	2023-09-20 20:28:11.297703
+4	Giraffe	Giraffa camelopardalis	111000	VU	2023-09-20 20:28:11.297703
+5	Polar Bear	Ursus maritimus	22000	VU	2023-09-20 20:28:11.297703
+6	Panda	Ailuropoda melanoleuca	1800	EN	2023-09-20 20:28:11.297703
+\.
+
+
+--
+-- Name: individuals_id_seq; Type: SEQUENCE SET; Schema: public; Owner: tpl1122_1
+--
+
+SELECT pg_catalog.setval('public.individuals_id_seq', 19, true);
```

Comment:
Why is this necessary? What happens if you miss it?

Reply:
This is an automated file that is made for me :D

Reply:
Yes, I know, but it's still an interesting line to know about. :)